packages/lwc/scripts/utils/rollup.js

```diff
@@ -54,8 +48,11 @@ function rollupConfig(config) {
             plugins: [
                 prod &&
                     rollupReplace({
-                        'process.env.NODE_ENV': JSON.stringify('production'),
+                        values: {
+                            'process.env.NODE_ENV': JSON.stringify('production'),
```

Comment:
SWC can also take care of doing the `process.env.NODE_ENV` replacement: https://swc.rs/docs/configuration/compilation#jsctransformoptimizerglobals. Here is the config I used locally.

```js
 transform: {
    optimizer: {
        globals: {
            envs: {
                "NODE_ENV": JSON.stringify('production')
            }
        }
    }
}
```

Reply:
Man, I could not find that anywhere! Thanks.

Reply:
Hm, actually this doesn't really work, at least with `swc.bundle()`. It just always takes whatever the actual NODE_ENV environment variable is. I tried many different ways to work around it (`mode: "none"`), `vars` instead of `envs`, but nothing works.

In fact it seems there's no way to _preserve_ `process.env.NODE_ENV` with swc, which is what some of our build outputs use, so I guess we should stick with Rollup for now.